examples/sites/src/views/components/components/anchor.vue

```
@@ -0,0 +1,116 @@
+      key: name,
+      link: `#cmp-${name}`,
+      title: name,
+      children: typeLinks
+    }
+    apiAnchorLinks.push(linkItem)
+  })
+
+  return apiAnchorLinks
+})
+const links = computed(() => (props.activeTab === 'demos' ? demoAnchorLinks.value : apiAnchorLinks.value))
+const emit = defineEmits(['link-click'])
```

ESLint

[error] defineEmits should be the first statement in `<script setup>` (after any potential import statements or type definitions).

(vue/define-macros-order)